### PR TITLE
Sprint 1/issues 2#implementing design

### LINKED ISSUE #2: sprint-1/issues-2#implement-design
## IMPLEMENT DESIGN 

> implement the design components of the application's registry design

### Tasks
- [x] Add icons
- [x] Add design components
- [x] Add change color validation in buttons

## DESIGN

> image

### PR DESCRIPTION
**Pull request by VISOSO2403**
# Description

What did you implemented/Why did you implemented this:
Sign up design completed. 
We added a library for the checkBoxes.
[react-native-bouncy-checkbox](https://github.com/WrathChaos/react-native-bouncy-checkbox)

We added the google icon and built the buttons.

We add the TouchableWithoutFeedback component to close the keyboard when pressing anywhere on the screen outside the input.

## Screenshots

image

### CODE REVIEW

**Review by Alfredo20164920 — APPROVED** [review]

File: ReservacionDeVuelos/src/components/index.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+
+export {
+  TextComponent,
+  ContainerComponent,
+  InputComponent,
+  ButtonComponent,
+  RowComponent,
+  CheckBoxComponent
+}
```
Comment (juperameza) [inline ReservacionDeVuelos/src/components/index.ts]: Add a new line at the end of this file

File: ReservacionDeVuelos/src/screens/index.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import SignUpScreen from './auth/SignUpScreen';
+
+export {
+  SignUpScreen
+}
```
Comment (juperameza) [inline ReservacionDeVuelos/src/screens/index.ts]: Add a new line at the end of this file

**Review by juperameza — CHANGES_REQUESTED** [review]

File: ReservacionDeVuelos/src/screens/index.ts (hunk 2 above)
Comment (VISOSO2403) [reply ReservacionDeVuelos/src/screens/index.ts]: ready :D

**Review by juperameza — APPROVED** [review]
Lgtm

**Pull request merged**